src/Extensions/DependencyInjection/FunctionHostBuilderContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Azure.Functions.Extensions.DependencyInjection
+{
+    public abstract class FunctionHostBuilderContext
+    {
+        private string _rootPath;
+
+        public string RootPath
+        {
+            get
+            {
+                if (_rootPath == null)
+                {
+                    string home = Environment.GetEnvironmentVariable("HOME");
+
+                    if (home != null)
+                    {
+                        _rootPath = Path.Combine(home, "site", "wwwroot");
+                    }
+                    else
+                    {
+                        _rootPath = Environment.GetEnvironmentVariable("AzureWebJobsScriptRoot");
+                    }
+                }
```

Comment:
Is this the same pattern as on Linux?

Reply:
Yes, it is

Reply:
Pretty sure I've validated, but it has been a while, @ahmelsayed can you confirm?

Reply:
Yes, that pattern is correct. $HOME on linux workers is /home, and the app content is in /home/site/wwwroot/

Reply:
After my PR here (https://github.com/Azure/azure-functions-host/pull/5991) -- I'll be able to pass this in directly, taking it from the options in the WebHost. So maybe we don't need this logic at all once I follow-up with that wire-up.